tests/template_tests/tests.py

```diff
@@ -100,3 +105,64 @@ def test_extends_generic_template(self):
         child = engines['django'].from_string(
             '{% extends parent %}{% block content %}child{% endblock %}')
         self.assertEqual(child.render({'parent': parent}), 'child')
+
+
+class VariableResolveLoggingTests(SimpleTestCase):
```

Comment:
might be nice to put this in a different file like `test_logging.py` or something